docs/RC/OnRCStatus/index.md

```diff
@@ -0,0 +1,66 @@
+## OnRCStatus
+
+Type
+: Notification
+
+Sender
+: SDL
+
+Purpose
+: Inform HMI and mobile application about changes in RC module allocation.
+
+!!! must
+
+
+### Notification
+
+SDL must send OnRCStatus notification when
+ * RC application registers with SDL
+ * a module is allocated to an application 
+ * a module is de-allocated from an application (the module is freed when application exits, disconnects, goes to HMI level NONE)
+ * when an application no longer has the right to control the module (for example, a policy update can revoke the application's access to a module)
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|appID|Integer|true|ID of selected application|
+|allocatedModules|[Common.ModuleData](/docs/Common/Structs/index.md)|true|Contains a list (zero or more) of module types that are allocated to the application|
+|freeModules|[Common.ModuleData](/docs/Common/Structs/index.md)|true|Contains a list (zero or more) of module types that are free to access for the application|
+
+### Sequence Diagrams
+
+|||
+OnRCStatus
+![OnRCStatus](assets/OnRCStatus.png)
```

Comment:
the notification send to hmi after app2 get control of module 2 has wrong contents. there shall be two notifications. 
(app1, allocatedModuels=[module1], freeModules=[])
(app2, allocatedModules=[module2], freeModules=[])

Reply:
@yang1070, fixed with https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/commit/94cbb5adc9c146a65788927aa11bbb889e96b0ab